### PR TITLE
Release/0.3.0

### PR DESCRIPTION
**Pull request by mscwilson**
We've added several configuration options to help you create your ideal events. It's now possible to enable anonymous tracking to anonymise important client-side or server-side user and session identifiers. Also, for mobile apps, choose whether or not to add the Screen and Application context entities to events.

### CHANGELOG

**New Features**  
Enable screen and application context on mobile (#27)  
Add anonymous tracking features (#16)

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "version": "0.2.0",
+    "version": "0.3.0",
```
Comment (matus-tomlein) [inline .vscode/launch.json:2]: Nit: I don't think this was referring to the tracker version :D

**Review by matus-tomlein — CHANGES_REQUESTED** [review]
Can you please start the example app on an iOS simulator? I think we are missing a version change in `Podfile.lock`.

File: .vscode/launch.json (hunk 1 above)
Comment (mscwilson) [reply .vscode/launch.json]: 🙈

**Review by matus-tomlein — APPROVED** [review]
LGTM!

**Pull request merged**